src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -537,9 +543,30 @@ void GraphingCalculator::DisplayGraphSettings()
     flyoutGraphSettings->ShowAt(GraphSettingsButton);
 }
 
+void CalculatorApp::GraphingCalculator::AddShadow()
```

Comment:
Really liking how this looks! I think a drop shadow to the trace value popup would also look good, but maybe we can do this later.

Reply:
I think this would be a good followup piece.  The popup is a rectangle, so themeshadow is a much easier way to get that to happen.  That said, I want to keep changes in this PR related to the trace pointer.